public/models/ReverseMovementGem.js

```diff
@@ -0,0 +1,42 @@
+if (typeof exports !== 'undefined' && typeof module !== 'undefined' && module.exports) {
```

Comment:
Not sure why these ones are also added as new files? That's kinda weird as they are already there.

Reply:
yeah  i am not sure why they are here and also I am not sure at this point which files should stay and which should go. I think it happened yesterday when I did pull and it pulled everything XD